plugin-server/src/worker/ingestion/event-pipeline/1-emitToBufferStep.ts

```diff
@@ -21,8 +20,12 @@ export async function emitToBufferStep(
     const person = await runner.hub.db.fetchPerson(event.team_id, event.distinct_id)
 
     if (shouldBuffer(runner.hub, event, person, event.team_id)) {
-        const processEventAt = DateTime.now().plus({ seconds: runner.hub.BUFFER_CONVERSION_SECONDS })
-        await runner.hub.db.addEventToBuffer(event, processEventAt)
+        const processEventAt = Date.now() + runner.hub.BUFFER_CONVERSION_SECONDS * 1000
```

Comment:
Part of my comment got left unaddressed: This is untested and should be.

Reply:
I thought a big part of the issue was that the previous implementation was a bit complex - happy to add a test but adding some milliseconds to `Date.now()` feels like something pretty standard to do.

Although sure, probably famous last words